gatsby-node.js

```diff
@@ -53,6 +58,10 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
           frontmatter.template === 'OverviewTemplate'
             ? `${frontmatter.path}/*`
             : undefined,
+        relatedResourceLimit: Math.max(
```

Comment:
Will this sent a request regardless of whether there are related resources to be requested? Shouldn't we not send out the request if the limit is 0?

Reply:
If by "request", you mean GraphQL request, then yes it will always execute. Since the GraphQL queries are static in Gatsby, unfortunately conditionally requesting fields from the schema are not really possible. This will still request data from the schema, but it means that if the limit is `0`, an empty array will be returned rather than the field being empty.

Reply:
One other thing to mention is that requests to Swiftype are only done at build time if the `BUILD_RELATED_CONTENT` environment variable is set, otherwise it just reads data from the `src/data/related-pages.json` file.

Reply:
ahhhh okay! thank you for clarifying that.